README.md

```diff
@@ -1,101 +1,38 @@
-# HTML & CSS3 & JavaScript Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+## **TODO LIST**
+A simple app to help you manage the things you have to do in a day.<br />
 
-## Set-up GitHub Actions
+## Built With
 
-Please do the following **steps in this order**:
+- HTML,CSS, JavaScript
+- Frameworks, Webpack
+- Github, Linters
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+## Live Demo
 
-Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+[Live Demo Link](https://ice949.github.io/Webpack-Setup/)
 
-![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
 
-## Set-up linters in your local env
+## Authors
 
-**Note**: The `npm` package manager is going to create a `node_modules` directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a [`.gitignore`](https://git-scm.com/docs/gitignore) file and add `node_modules` to it:
+👤 **Justine Imasiku**
 
-```
-# .gitignore
-node_modules/
-```
+- GitHub: [@ice949](https://github.com/ice949)
+- Twitter: [@justine_imasiku](https://twitter.com/justine_imasiku )
+- LinkedIn: [Justine Imasiku](https://www.linkedin.com/in/justine-imasiku-7a25881a5/)
 
-### [Lighthouse](https://developers.google.com/web/tools/lighthouse)
+## 🤝 Contributing
 
-An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more.
+Contributions, issues, and feature requests are welcome!
 
-You can get the Lighthouse report by any of the following ways:
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] Kindly include commands for cloning, installing and starting this project on the development server. This will assist people to know how to clone and start your application without hassle.

_**Hint:**_
```readme.md
git clone <url>
cd <name of repository>
npm install
npm start
```